lamindb/models/save.py

```diff
@@ -157,7 +157,10 @@ def check_and_attempt_upload(
             return exception
         # copies (if on-disk) or moves the temporary file (if in-memory) to the cache
         if os.getenv("LAMINDB_MULTI_INSTANCE") is None:
-            copy_or_move_to_cache(artifact, storage_path, cache_path)
+            try:
```

Comment:
Please add a comment here because it looks problematic to add a blank try & except.

Otherwise fine for me.

Reply:
i will add.